src/Rejuvena.Collate.CLI/Commands/PackageModCommand.cs

```diff
@@ -122,6 +125,8 @@ public sealed class PackageModCommand : VersionSensitiveCommand, IPropertiesProv
         else await console.Output.WriteLineAsync("Could not resolve mod .pdb, expected at: " + modDll.Path);
 
         TModPacker.PackMod(options);
+
+        ModsFolderUtils.AddBuildPath(Path.Combine(Path.GetDirectoryName(OutputTmodPath)!, "collate.json"), AssemblyName, Path.Combine(ProjectDirectory, ProjectName) + ".csproj");
```

Comment:
Will need to keep a note that we assume the extension is `.csproj`, though AFAIK there's no way to guarantee this.